### ISSUE TITLE
keep constant vibration while the vive controller trigger was held

### ISSUE THREAD

**Issue by westside** [bhaptics/tac-sharp#37]
I had no problem creating and firing patterns, the issues occurred when I was trying to get the motors to fire constantly and increase and decrease in intensity. The vibration wasn't very consistent and whenever the intensity was decreased it cut out completely. 

Sorry, it's been a while since I looked at any of this, but I tried a couple of ways of achieving this but failed to find a satisfactory solution. I only have the final attempt which was submitting a byte array and charging the intensity values. 

Basically I was trying to keep constant vibration while the vive controller trigger was held, and adjust the intensity based on how far the trigger was pulled. I couldn't find any examples of how you could do this in you example or docs, though I might have missed something.

**Comment by westside** [comment]
* We can use ScaleOption and IsPlaying
   * you can control duration and intensity of feedback file with ScaleOption
   * By checking IsPlaying, whole feedback can be played.  

```
using Bhaptics.Tac;
using Bhaptics.Tac.Sender;
using Bhaptics.Tac.Unity;
using UnityEngine;

public class InputHandler : MonoBehaviour
{

    private SteamVR_TrackedObject controller;
    private IHapticPlayer hapticPlayer;

	// Use this for initialization
	void Start ()
	{
	    controller = GetComponent<SteamVR_TrackedObject>();
	    hapticPlayer = BhapticsManager.HapticPlayer;

	}
	
	// Update is called once per frame
	void Update () {
	    if (controller != null)
	    {
	        var steamDevice = SteamVR_Controller.Input((int)controller.index);
	        var triggerAxis = steamDevice.GetAxis(Valve.VR.EVRButtonId.k_EButton_SteamVR_Trigger).x;
           
	        var motorValue = (byte)((Mathf.Max(triggerAxis, 0f)) * 100);
	        var bytes = new byte[20];

	        for (int i = 0; i < 20; i++)
	        {
	            bytes[i] = motorValue;
	        }
	        Debug.Log("val " + motorValue);
	        var isRight = gameObject.name.Contains("right");
//            hapticPlayer.Submit("test", gameObject.name.Contains("right") ? PositionType.Right : PositionType.Left, bytes, 100);

	        if (!hapticPlayer.IsPlaying("BowShoot"))
	        {
	            hapticPlayer.SubmitRegistered("BowShoot", new ScaleOption(triggerAxis, 0.5f));
            }
	        
	    }
	}
}
```

**Comment by westside** [comment]
* This is an example project
[example package link](https://drive.google.com/file/d/1PvHgtysNsAc-wzQBNRXSz0PSx5WYT7KR/view?usp=sharing)
